### PR TITLE
Poke Battle Added

### PR DESCRIPTION
**Pull request by bwallick**

### CODE REVIEW

**Comment by willpost1292** [comment]
Yo Blake, just a note on your pokemon battle simulator so far...

*Cool while loop w/ pokemon == null very logical

*I see that you put the Pokemon options in that while loop, and then your calling on the
methods choosePokemon() that you havn't fleshed out yet. (lines 41- 42)

*I would put that while loop inside the choose Pokemon() method, something like:

main():
  print "Welcome to Pokemon Arena!" 
  print "player one, please enter your pokemon"

  p1 = choosePokemon()

  print "player two, please enter your Pokemon:"

  p2 = choosePokemon()

  battle(p1,p2);

//and then have:

public pokemon choosePokemon(Scanner user in){          
//you have a Scanner as a param here, what's that for? J/W
  
  Pokemon newPoke = null;
  
  while(newPoke == null):
     
     //and have the options here
     

TLDR: It's good modular programming to have choices and the user selection in the same method, 
          that way if you ever change the choices, you can change them in one place, etc
